cli.js

```diff
@@ -113,6 +139,13 @@ if (cli.input.length === 0) {
 	const promise = fkill(cli.input, Object.assign(cli.flags, {ignoreCase: true}));
 
 	if (!cli.flags.force) {
-		promise.catch(() => handleFkillError(cli.input));
+		promise.catch(err => {
+			if (/Couldn't find a process with port/.test(err.message)) {
```

Comment:
I think you meant `if (/Process doesn't exist/.test(err.message)) {` here ;)

Reply:
I can add that too, but in this case we get the `Couldn't find a process with port` from the `pid-from-port` module.

Reply:
Weird, it works now. Not sure what changed...